earlydisplay/src/main/java/net/neoforged/fml/earlydisplay/DisplayWindow.java

```diff
@@ -165,10 +164,12 @@ public void initialize(ProgramArgs arguments) {
         if (Boolean.getBoolean("fml.earlyWindowDarkMode")) {
             this.darkMode = true;
         } else {
-            try {
-                var optionLines = Files.readAllLines(FMLPaths.GAMEDIR.get().resolve(Paths.get("options.txt")));
-                var options = optionLines.stream().map(l -> l.split(":")).filter(a -> a.length == 2).collect(Collectors.toMap(a -> a[0], a -> a[1]));
-                this.darkMode = Boolean.parseBoolean(options.getOrDefault("darkMojangStudiosBackground", "false"));
+            try (var lines = Files.lines(FMLPaths.GAMEDIR.get().resolve(Paths.get("options.txt")))) {
+                this.darkMode = lines
+                        .filter(l -> l.startsWith("darkMojangStudiosBackground"))
```

Comment:
Small nit: checking that it starts with `darkMojangStudiosBackground:` (colon at the end) would handle the case that a mod decided to add their own option with a different name that still starts with `darkMojangStudiosBackground`.